packages/admin/src/brevoContacts/form/BrevoContactForm.tsx

```diff
@@ -76,11 +77,15 @@ export function BrevoContactForm({ id, scope, input2State, additionalFormFields,
         id ? { variables: { id, scope } } : { skip: true },
     );
 
-    const initialValues = React.useMemo<Partial<EditBrevoContactFormValues>>(() => {
+    const initialValues = React.useMemo<Partial<BaseBrevoContactFormValues>>(() => {
         let additionalInitialValues = {};
 
         if (input2State) {
-            additionalInitialValues = input2State({ email: "", redirectionUrl: "", ...data?.brevoContact });
+            additionalInitialValues = input2State({
+                email: "",
+                redirectionUrl: "",
```

Comment:
these values should be in the `additionalInitialValues` (would rename to `baseInitalValues`). Since they are not needed in the `input2State` anymore. And the `input2state` values should be added to these `baseInitalValues`